docs/EN/Usage/Smoothing-Blood-Glucose-Data.md

```diff
@@ -1,25 +1,25 @@
 (Smoothing-Blood-Glucose-Data)=
 # Smoothing blood glucose data
 
-If **BG** data is jumpy/noisy, AAPS may dose insulin incorrectly resulting in highs or lows. If you observe errors in your CGM data it is important to disable the loop until the problem is resolved. Depending on your CGM, such issues may be due to the CGM configuration in **AAPS** (as explained further below); or CGM sensor site issue (which may require replacing the CGM sensor)
+If **BG** data is jumpy/noisy, **AAPS** may dose insulin incorrectly resulting in highs or lows. If you observe errors in your CGM data it is important to disable the loop until the problem is resolved. Depending on your CGM, such issues may be due to the CGM configuration in **AAPS** (as explained further below); or a CGM sensor site issue (which may require replacing the CGM sensor).
 
-Some CGM systems have internal algorithms to detect the noise level in the readings and AAPS can use this information to avoid giving SMBs if the BG data is too unreliable. However, some CGMs do not transmit this data and for these BG sources 'Enable SMB always' and 'Enable SMB after carbs' are disabled for safety reasons.
+Some CGM systems have internal algorithms to detect the noise level in the readings, and **AAPS** can use this information to avoid giving SMBs if the BG data is too unreliable. However, some CGMs do not transmit this data and for these BG sources 'Enable SMB always' and 'Enable SMB after carbs' are disabled for safety reasons.
 
-Additionally, as of version 3.2, AAPS offers the option to smooth the data within AAPS. There are three options available in the [Config Builder](../Configuration/Config-Builder.md).
+Additionally, as of **AAPS** version 3.2, **AAPS** offers the option to smooth the data within **AAPS** rather than within the CGM app. There are three options available in the [Config Builder](../Configuration/Config-Builder.md).
 
 ![Smoothing](../images/ConfBuild_Smoothing.png)
 
 ## Exponential smoothing
 
-This is the recommended option to start with as it is most aggressive in resolving noise and rewrites the most recent value.
+This is the recommended option to start with, as it is most aggressive in resolving noise and rewrites the most recent value.
```

Comment:
Maybe expand the recommendation to : This is recommended when you have noisy data. Also note that exponential smoothing adds lag

Reply:
OK will do that in the next revision, thanks.